public/scripts/components/pages/Read.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default React.createClass({
+	render: function() {
+		return (
+			<section className="read-story">
+				<h1>Read Story</h1>
+			</section>
+		);
+	}
+});
```

Comment:
You shouldn't be changing the Read component for this ticket. Why does it look like this file is modified?